porch/controllers/rootsyncset/pkg/controllers/rootsyncset/controller.go

```diff
@@ -58,6 +57,11 @@ type RootSyncSetReconciler struct {
 //+kubebuilder:rbac:groups=config.porch.kpt.dev,resources=rootsyncsets,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=config.porch.kpt.dev,resources=rootsyncsets/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=config.porch.kpt.dev,resources=rootsyncsets/finalizers,verbs=update
+//+kubebuilder:rbac:groups=configcontroller.cnrm.cloud.google.com,resources=configcontrollerinstances,verbs=get;list
```

Comment:
I always add in watch, personally, even if we're not using it.  My 2c is that unless we're locking it down to one object by name, we should just request get,list,watch for read-only.

Reply:
I agree. I have updated the PR.